CHANGES.rst

```diff
@@ -1,16 +1,11 @@
-0.9.0 (unreleased)
-==================
-
-- Add ``resource_tracker`` and ``log_tracked_resources`` fixtures. [#74]
```

Comment:
This removal does not look right.

Reply:
I moved this entry to `changes/74.misc.rst`

Reply:
Ah, so when will changelog be updated with that fragment?

Reply:
on running `towncrier build` on the next release (it does have to be a manual process with a commit as part of the release process)